framework/Simulation.py

```diff
@@ -790,3 +794,26 @@ def generateAllAssemblers(self, objectInstance):
                                       .format(n=obj[1], m=mainClassStr) +
                                       '\nOptions are:', self.whichDict[mainClassStr].keys())
       objectInstance.generateAssembler(neededobjs)
+
+  def clearStatusFile(self):
+    """
+      A bad hack from ancient technologies so we can really tell
```

Comment:
:) Can you change the the docstring with something more "neutral" and descriptive :D

Reply:
political correctness employed :P